.changeset/ninety-dogs-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Now avatar component in staff user details display initials and actions in white color with proper font styles. 
```

Comment:
_Now you can see initials in your account details when there is no avatar set for your user account_

change you make influences the user directly and what he sees